views/js/runner/provider/qti.js

```diff
@@ -256,10 +271,21 @@ define([
                                     context.itemAnswered = result.itemSession.itemAnswered;
                                 }
 
-                                if(result.displayFeedbacks === true && itemRunner){
-                                    return itemRunner.trigger('feedback', result.feedbacks, result.itemSession, resolve);
+                                if(result.displayFeedbacks === true && result.feedbacks && result.itemSession){
+
+                                    itemRunner.renderFeedbacks(result.feedbacks, result.itemSession, function(queue){
+                                        self
```

Comment:
Try to avoid calling plugin events. 

You can give a callback to the modalFeedback event: 

```js
self.trigger('modalFeedbacks', queue, resolve);
```

and in the plugin 

```js
this.getTestRunner().on('modalFeedback', function(queue, done){

});
```